### ISSUE TITLE
[Suggestion] One text file with all rules together

### ISSUE THREAD

**Issue by JobLeonard** [uMatrix-Rules/uMatrix-Rules.github.io#5]
I know this is a website that tries to combine the recipes from a lot of different sources. I just manually added everything to my uMatrix, and by god does the internet suddenly run fast! I really appreciate your effort.

Anyway, I think that for many less tech-savvy people the switch to uMatrix would be a lot less intimidating if there was one big default file that combines all of these recipes, or maybe a few with different levels of "strictness"

Again, thanks for making these rules! I can't believe my internet is so fast again! :)

**Comment by Riajyuu** [comment]
If you have read my docs carefully enough, you will find that there are nearly just the same. They just different rulesets in different occasions.

**Comment by JobLeonard** [comment]
Yes, it's a good documentation site!

But my point was about a quick installation for users who don't want to spend too much time setting up uMatrix.

Loading one text file with presets for a lot of sites saves them the hassle of doing this manually for each, and you already have most of the work done for that!

I found your repo through the [default recipes from `uMatrix Recipes`](https://github.com/kristerkari/umatrix-recipes), and you also have [another repo with a single text file for the Alexa 100](https://github.com/uMatrix-Rules/uMatrix-Rules-Domain). 

Combine both in one big text file, link it from the umatrix-rules website with an explanation of how to load/export settings. Ideally with a screenshot of the settings page, like the one below but with the normal theming for Firefox, since mine is a little unusual.

![image](https://user-images.githubusercontent.com/259840/42224129-9f1e85f2-7ed9-11e8-94d2-0d6fc99b8ab6.png)

**Comment by Riajyuu** [comment]
@JobLeonard That repository is from one our members who joined recently @kristerkari. He hasn't finished things yet.

**Comment by JobLeonard** [comment]
I will be patient then :)

**Comment by kristerkari** [comment]
Yeah sorry, I have been busy with other things, so I haven't done anything yet :)

**Comment by JobLeonard** [comment]
No worries, personally I'm already grateful for the collections you all have.
